designs/DotNetSignCommands.md

```diff
@@ -0,0 +1,140 @@
+# Spec Name
+
+- Status: In Review
+- Author(s): [Kat Marchán](https://github.com/zkat)
+- Issue: [9010](https://github.com/NuGet/Home/issues/9010) Signing: Design dotnet package signing commands
+
+## Problem Background
+
+We recently released [cross-platform signature verification during
+restore](#todo) for `dotnet.exe`. This means `dotnet.exe` users are going to
+start relying on this feature during restore for security. Unfortunately,
+`nuget.exe` has some signing-related commands that are not yet available in
+`dotnet.exe`, making the experience essentially incomplete for these
+customers.
+
+## Who are the customers
+
+This feature is for `dotnet.exe` users who want to take advantage of more
+package signing-related features of nuget without needing to install
+`nuget.exe`.
+
+## Goals
+
+Design and implement equivalent commands for the following `nuget.exe` commands:
+
+- `nuget.exe sign`
+- `nuget.exe verify`
+- `nuget.exe trusted-signers`
+
+These commands should have `dotnet.exe`-compatible syntax and ergonomics, and
+feel like proper `dotnet.exe` commands.
+
+## Non-Goals
+
+Adding additional signing-related commands, or significantly
+changing/improving the behavior of the commands as currently designed.
+
+## Solution
+
+The following commands will be implemented in the `dotnet.exe` CLI.
+
+Options marked with `*` indicate a notable change in behavior/naming.
+
+### `dotnet nuget sign`
+
+```
+Signs a NuGet package at <package-path> with the specified certificate.
+
+USAGE:
+    dotnet nuget sign [OPTIONS] <package-path>
+
+OPTIONS:
+        --output-directory              Directory where the signed package should be saved. By default, the original package is overwritten by the signed package.
```

Comment:
dotnet CLI has a tendency to do `[-o|--output <OUTPUT_DIRECTORY>]`. Do we want to follow that here?